Adds useful links/resources to dev docs smart-contract/security page (#12286)
ethereum/ethereum-org-website: public/content/developers/docs/smart-contracts/security/index.md

- Use a [development environment](/developers/docs/frameworks/) for testing, compiling, deploying smart contracts

> - Run your code through basic code analysis tools, such as, [Cyfrin Aaderyn](https://cyfrin.io/tools/aderyn), Mythril and Slither. Ideally, you should do this before each pull request is merged and compare differences in output

Reviewer: Proposed link is redirecting to https://github.com/Cyfrin/aderyn

```suggestion
- Run your code through basic code analysis tools, such as, [Cyfrin Aaderyn](https://github.com/Cyfrin/aderyn), Mythril and Slither. Ideally, you should do this before each pull request is merged and compare differences in output
```

- **[CertiK](https://www.certik.com/)** - _Blockchain security firm pioneering the use of cutting-edge formal Verification technology on smart contracts and blockchain networks._

> - **[Cyfrin](https://cyfrin.com)** - _Industry-leading Smart contract security auditing, educatio and tools. Helping protect the world's biggest protocols.
>   

Reviewer: Already added in #11182

```suggestion
```

### Tutorials on smart contract security {#tutorials-on-smart-contract-security}

> - [Cyfrin Updraft - Smart contracts security and Auditing full course] (https://updraft.cyfrin.io/courses/security)

Reviewer: ```suggestion
- [Cyfrin Updraft - Smart contracts security and auditing full course] (https://updraft.cyfrin.io/courses/security)
```
